jcl/src/java.base/share/classes/java/lang/StackWalker.java

```diff
@@ -55,9 +55,14 @@
 public final class StackWalker {
 
 	private static final int DEFAULT_BUFFER_SIZE = 1;
```

Comment:
It would be helpful to have a comment here noting that these constants must be consistent with like-named macros in `java_lang_StackWalker.cpp`.

Reply:
Added comment, also updated to use hex value to show constants used as bit flag